krux/cli.py

```diff
@@ -139,35 +140,53 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
         if lockfile:
             self.acquire_lock(lockfile)
 
+    def init_logging(self):
+        ### get a logger - is there a way to to have logger be relative to the
+        ### invocation of the log call?
+        self.logger = self.logger or krux.logging.get_logger(
+            self.name,
+            level=self.args.log_level
+        )
+        if self.args.log_file is not None:
+            ### TODO: More of this should be configurable
+            handler = logging.handlers.TimedRotatingFileHandler(
+                self.args.log_file,
+                when='midnight'
```

Comment:
please let's not add strings in the code; let's make this a constant at the top, and potentially even something that can be overridden.